managedtenants/schemas/metadata.schema.yaml

```diff
@@ -143,7 +143,8 @@ properties:
           description: "Name of the additional catalog source."
         image:
           type: string
-          description: "Url of the additional catalog source image."
+          pattern: ([\w\-]+[.][\w\-.]+([:][0-9]+)?\/)([\w\-]+\/)?[a-zA-Z0-9][\w\-.]+[a-zA-Z0-9](@sha256:[0-9a-f]{64})$
```

Comment:
done @apahim! PTAL at the pattern. I took some help from here: https://github.com/app-sre/sretoolbox/blob/97cc4db175086584cf2e9856bb58dfbba1e8aa57/sretoolbox/container/image.py#L282-L300

Reply:
Hi @rpriyanshu9, nice that you knew about that regex! It's super complex, because it's meant to be a parser, to isolale each component of the URL.

I believe we need something very simplistic here, just to make sure it's a digest, but also easy to maintain.

I think it would be enough to have something like ".*@sha256:.*". What do you think? I know, it must have tekaen you quite some time to come up with that regex, but that's exactly my point :D

Reply:
Hey, thanks for your feedback. I agree with your point. What do you think about this, `.*@sha256:[0-9a-f]{64}$`?

Reply:
- https://github.com/mt-sre/managed-tenants-cli/blob/81f5ab059ca9e692121f072f6bf0e4767e823800/tests/testdata/addons/mock-operator-with-secrets/metadata/stage/addon.yaml#L47
- https://github.com/mt-sre/managed-tenants-cli/blob/81f5ab059ca9e692121f072f6bf0e4767e823800/tests/testdata/addons/test-operator/metadata/integration/addon.yaml#L22

Reply:
Fair enough :+1: